ocp_resources/node.py

```diff
@@ -1,16 +1,130 @@
+# Generated using https://github.com/RedHatQE/openshift-python-wrapper/blob/main/scripts/resource/README.md
+
+from typing import Any, Dict, Optional
 from ocp_resources.resource import Resource
 
 
 class Node(Resource):
```

Comment:
Please generate the resource with the latest script code.

Reply:
done